### PR TITLE
Add a script for importing VSP certificates into the Key Vault

### PR DESCRIPTION
**Pull request by erbridge**
This has already been run on the certificates we just received back from Verify.

### CODE REVIEW

**Comment by erbridge** [comment]
[Get "Integration" (i.e. staging) Verify certificates for running a VSP](https://trello.com/c/FYRKfGar/345-get-integration-ie-staging-verify-certificates-for-running-a-vsp)

File: bin/import-vsp-certs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+function store_certificate {
+  local FILE_PREFIX=$1
+  local CERTIFICATE=$2
+
+  if ! [ "$CERT_EXISTED" ]; then
+    az keyvault secret set \
+      --vault-name $KEY_VAULT_NAME \
+      --name "${FILE_PREFIX}Certificate" \
+      --value "$CERTIFICATE"
```
Comment (pezholio) [inline bin/import-vsp-certs]: Does this work OK with keyvault? I had to use the `--file` option as the linebreaks got mangled if I set a value

Comment (erbridge) [reply bin/import-vsp-certs]: Yes, it does. Just confirmed by looking at what's in the Key Vault, imported by this script.

Comment (pezholio) [reply bin/import-vsp-certs]: 🆒

**Comment by pezholio** [comment]
We're using a lot of shell scripts now, do you think it's worth adding `shellcheck` to our CI?

**Comment by erbridge** [comment]
Possibly. I have it integrated into my editor, but yes. Add a card to the backlog?

**Comment by pezholio** [comment]
Yeah, it might be better as a seperate story

**Comment by erbridge** [comment]
`shellcheck` card added to the backlog.

**Pull request merged**